charts/cluster-autoscaler-chart/values.yaml

```diff
@@ -150,6 +150,9 @@ extraArgs:
 # extraEnv -- Additional container environment variables.
 extraEnv: {}
 
+# extraVolumeSecrets -- Additional volumes to mount from Secrets.
```

Comment:
Can you include a full example of using this commented out given there's a number of sub-options here please.